tests/rptest/wasm/wasm_test.py

```diff
@@ -165,25 +165,29 @@ def accrue(acc, output_topic):
         self.logger.info(
             f"Waiting for {total_inputs} input records and {total_outputs}"
             " result records")
-        self._input_consumer.start()
-        self._output_consumer.start()
```

Comment:
can you describe in the commit message what is solved by not starting the consumers concurrently? generally it sounds like a concurrency bug is being avoided without extra context.

Reply:
removed this commit